src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/WidgetsStyles.kt

```diff
@@ -148,22 +148,22 @@ class WidgetsStyles : Stylesheet() {
         }
 
         wizardCard {
-            maxHeight = 364.0.px
-            prefHeight = 364.0.px
-            prefWidth  = 364.0.px
+            maxHeight = 320.0.px
+            prefHeight = 320.0.px
+            prefWidth  = 300.0.px
             backgroundRadius += box(12.0.px)
-            backgroundColor += c("#CC4141")
-            textFill = c("#FFFF")
+            backgroundColor += c("#FFFF")
+            textFill = c("#CC4141")
             fontSize = 24.px
             effect = DropShadow(10.0, Color.GRAY)
-            cursor = Cursor.HAND
         }
 
         wizardCardButton {
             prefHeight = 40.0.px
-            prefWidth = 164.0.px
-            backgroundColor += c("#FFFF")
-            textFill = c("#CC4141")
+            prefWidth = 250.0.px
+            backgroundColor += c("#CC4141")
+            textFill = c("#FFFF")
```

Comment:
Use colors object for these?

Reply:
We don't want our widgets to be dependent on color resources. They need to be separate from each other